src/storage/src/hummock/compactor/mod.rs

```diff
@@ -247,6 +246,18 @@ impl Compactor {
         // Sort by split/key range index.
         output_ssts.sort_by_key(|(split_index, ..)| *split_index);
 
+        {
+            // cancel space_reclaim_compaction when delete ratio too low
+            if output_ssts.is_empty()
```

Comment:
Is it possible "output_ssts.is_empty()" being the result of successful space_reclaim_compaction? e.g. task input's all KVs belong to dropped tables thus they are all deleted.

Reply:
Good catch, I need to distinguish between two states
- All files are cleaned up
- The delete ratio is too low and no need to perform this task to rewrite